third_party/musl/src/starboard/network/socket.c

```diff
@@ -350,6 +357,7 @@ int open(const char* path, int oflag, ...) {
     mode_t mode = va_arg(args, int);
     if (mode != sb_file_mode) {
       out_error = kSbFileErrorFailed;
+      errno = EINVAL;
```

Comment:
Was it problematic when we did not set errno here?

Reply:
Yes. It would cause the calling code to be confused about `errno` being zero, even though the function failed.